pkgs/development/arduino/arduino-core/default.nix

```diff
@@ -101,13 +174,21 @@ stdenv.mkDerivation rec {
     # avrdude_bin is linked against libtinfo.so.5
     mkdir $out/lib/
     ln -s ${lib.makeLibraryPath [ncurses5]}/libncursesw.so.5 $out/lib/libtinfo.so.5
+
+    ${stdenv.lib.optionalString (withTeensyduino == true) "
```

Comment:
Same as above - I think it would look nicer if it wasn't inlined but (conditionally) appended